content/en/real_user_monitoring/dashboards/testing_coverage.md

```diff
@@ -0,0 +1,53 @@
+---
+title: RUM Application Testing Coverage Dashboard
```

Comment:
I believe we can drop the Synthetics or RUM in the title and stay with "Application Testing Coverage Dashboard"